CHANGELOG.md

```diff
@@ -240,7 +241,7 @@ Improvements:
 
 ## [0.1.0] - 2019-05-14
 
-[Unreleased]: https://github.com/Componolit/RecordFlux/compare/v0.5.0...HEAD
+[unreleased]: https://github.com/Componolit/RecordFlux/compare/v0.5.0...HEAD
```

Comment:
What is the reason for this change? This change was not there before you squashed the commits.

Reply:
I used `prettier` on the changelog, and unfortunately this is a known bug: https://github.com/prettier/prettier/issues/7118. Will revert on this one.

Reply:
Done.

Reply:
I see.